src/api/v1/statuses.ts

```diff
@@ -276,6 +277,26 @@ app.post(
         preferSharedInbox: true,
         excludeBaseUris: [new URL(c.req.url)],
       });
+      if (!owner.account.protected) {
+        const acceptedRelays = await db.query.relays.findMany({
+          where: eq(relays.state, "accepted"),
+          with: {
+            relayServerActor: true,
+          },
+        });
+        await fedCtx.sendActivity(
+          { handle },
+          acceptedRelays.map((relay) => ({
+            id: new URL(relay.relayServerActor.iri),
+            inboxId: new URL(relay.relayServerActor.inboxUrl),
+          })),
+          activity,
+          {
+            preferSharedInbox: true,
```

Comment:
We don't have to turn this on as we aren't providing recipients' shared inbox URLs here.